HW8/src/ArrayValueCalculator.java

```diff
@@ -0,0 +1,46 @@
+import exceptions.ArrayDataException;
+import exceptions.ArraySizeException;
+
+public class ArrayValueCalculator {
+
+    public static void main(String[] args) throws ArrayDataException, ArraySizeException {
+
+        String[][] arrayForCalc = new String[][] {
+            {"21", "()*)", "33", "41"},
+            {"35", "**", "27", "18"},
+            {"19", "10", "11", "*913"},
+            {"9", "10", "11", "12"}
+        };
+
+        int dimension = 4;
+
+        System.out.println("\nResult of doCalc():");
+        System.out.println(doCalc(arrayForCalc, dimension));
+    }
+
+    private static int doCalc(String[][] arrayForCalc, int dimension) throws ArraySizeException, ArrayDataException {
```

Comment:
Must be public because this is the method that actually does a logic for anyone

Reply:
Fixed according to comments, tested and merged